include/git2/common.h

```diff
@@ -270,6 +271,18 @@ typedef enum {
  *		> example, when this is enabled, the parent(s) and tree inputs
  *		> will be validated when creating a new commit.  This defaults
  *		> to disabled.
+ *
+ *	* opts(GIT_OPT_ENABLE_SYMBOLIC_REF_TARGET_VALIDATION, int enabled)
+ *
+ *		> Validate the target of a symbolic ref when creating it.
+ *		> For example, 'foobar' is not a valid ref,
+ *		> therefore 'foobar' is not a valid target
+ *		> for a symbolic ref by default,
+ *		> where as 'refs/heads/foobar' is.
```

Comment:
Could you change these to backticks for readability in the documentation?  Also, one nitpick: `whereas` should be one word.